README.md

```diff
@@ -63,6 +63,18 @@ repository.
 If you encounter any issues whilst using cert-manager, and your issue is not
 documented, please [file an issue](https://github.com/jetstack/cert-manager/issues).
 
+Possible resources for *help*:
+
+* The official channel `#cert-manager` on `kubernetes.slack.com`
+
+> There is also a good chance to get some support on non-official support
+> channels for *cert-manager*, but be aware that these are rather general
+> kubernetes discussion channels.
+
+* `#coreos` on freenode
+* Slack channels like `#kubernetes-users` or `#kubernetes-novice` on `kubernetes.slack.com`
+* If you absolutely just can't figure out your problem, file an issue.
```

Comment:
Thanks for this!

I think we can remove the IRC support reference (afaik, nobody from Jetstack and myself personally do not use it!)

Also, perhaps we could reword the final point (filing an issue) to not sound like we *discourage* filing an issue. Perhaps something more like `If you think you've run into a bug, and you cannot find an existing issue open that describes your problem, please file an issue against the repo [here](https://github.com/jetstack/cert-manager/issues).`?